examples/marketplace.jsligo

```diff
@@ -1,11 +1,17 @@
-#import "../lib/fa2/nft/nft.impl.jsligo" "Contract"
+#import "../lib/main.mligo" "FA2"
+
+import Contract = FA2.NFT
+
+// FIXME These files make no sense and should probably be removed
```

Comment:
Why ? It's an example and you are pointing it into the readme file.

Reply:
The code is ugly and the example adds nothing. This is fixed in the next PR, where I give a better example. I can remove the comment though, but I think it's clear to the reader that the example is a WIP, no?